wildmatch.go

```diff
@@ -69,6 +83,32 @@ const (
 	escapes = "\\[]*?"
 )
 
+// pathname converts patterns of the form "foo*.txt" to "foo/**/*.txt", thus
```

Comment:
Should `foo*.txt` be converted? I couldn't find any documentation on Git's Wildmatch code, but I know with a unix file glob `foo*.txt` would match `foo.txt`, `fooabc.txt`, etc., but not `foo/abc.txt`.